### ISSUE TITLE
transition is not selectable if it is drawn over state

### ISSUE THREAD

**Issue by franzlst** [DLR-RM/RAFCON#501]
Transition is not selectable if it is drawn over state. Via points cannot be grasped or created.

As discussed in:
https://wiki.robotic.dlr.de/RAFCON/Meetings#2018-03-15

Originally created by @sebastian-brunner  (Sebastian.Brunner@dlr.de) at 2018-03-20 08:24:23+00:00 (moved from RMC internal repository)

**Comment by franzlst** [comment]
The source of this issue is that the transition can not be hovered in the area where it is drawn over a state.

Thereby this is an issue I also discussed with @franzlst before this meeting and the outcome was that he was seeing it as a very minor issue.

At that time, 2 weeks ago, I was annoyed because I can not add or move the via-points of the transition/via-points (possible future via-points) drawn over a state.

Sorry for my late notes on this, in the meeting I was maybe to much into the installation problems of RAFCON what we tried to figure out in parallel.

Originally created by @Rbelder  at 2018-03-20 10:29:05+00:00 (moved from RMC internal repository)
